v2/cell.go

```diff
@@ -0,0 +1,313 @@
+package chronograf
```

Comment:
What do you think about just calling this platform?  I imagine that'll make it slightly easier when we move it.